doc/api/policy.md

```diff
@@ -0,0 +1,42 @@
+# Policies
+
+<!--introduced_in=TODO-->
+<!-- type=misc -->
+
+> Stability: 0 - Experimental
+
+<!-- name=policy -->
+
+Node.js contains experimental support for creating policies on loading code.
+
+## Enabling
+
+<!-- type=misc -->
+
+The `--experimental-policy` flag can be used to enable features for policies when loading modules.
+
+Once this has been set, all modules must conform to a policy file passed to the flag:
+
+```sh
+node --experimental-policy policy.json app.js
```

Comment:
I'd prefer syntax like `--experimental-policy=policy.json app.js` (note the `=`)

Reply:
this doesn't mesh with other cli options like `--require`

Reply:
@jasnell @bmeck Fun fact: Because we were previously inconsistent about this w.r.t. different options, *both* work with the new options parser. Like, you can also currently use `--require=...` instead of `--require …`, if you prefer :)

Reply:
changed doc, though autocomplete won't work with the = form, curious why the preference but not really anything of note. I do have some concerns and need to think of if we ever want separate flags for policies if this occupies the = space.